web/src/message_edit_history.js

```diff
@@ -190,7 +190,7 @@ export function initialize() {
         }
 
         if (page_params.realm_allow_edit_history) {
-            show_history(message);
+            fetch_and_render_message_history(message);
             $("#message-history-cancel").trigger("focus");
```

Comment:
I checked that this `trigger` code is not needed anymore as there is no such element and focus is handled by passing `focus_on_submit` as true to `dialog_widget`, so we can remove this in a separate commit.